tests/examples/test_simulate_single_robot.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from ..conftest import EXAMPLES_DIR, assert_command_succeeds
+
+
+def test_simulate_single_robot() -> None:
+    """Test simulate_single_robot example can complete."""
+    exp_dir = os.path.join(EXAMPLES_DIR, "simulate_single_robot")
+    assert_command_succeeds(
+        [
+            "python",
+            os.path.join(exp_dir, "main.py"),
+            "-t",
```

Comment:
I don't think this flag still exists. However, speeding up sim to 1 seconds(i think that is what is intended here) is nice. maybe patch the `make_standard_batch_parameters`?

Reply:
im not sure how to approach this. wouldnt i have to pass something to the script?